test/prototype/mx_formats/test_inference_workflow.py

```diff
@@ -100,6 +101,16 @@ def test_inference_workflow_mx(elem_dtype, bias: bool, compile: bool, emulate: b
         f"Got a sqnr of {sqnr} for {elem_dtype} and bias={bias}"
     )
 
+    # serialization
+    with tempfile.NamedTemporaryFile() as f:
+        torch.save(m_mx.state_dict(), f)
+        f.seek(0)
+
+        # temporary workaround for https://github.com/pytorch/ao/issues/3077
+        torch.serialization.add_safe_globals([getattr])
```

Comment:
See my comment on this issue, let me know if it works for this case: https://github.com/pytorch/ao/issues/3077#issuecomment-3339670903

Reply:
the code in inference_workflow already does that :)

Reply:
Oh, I needed to expend/open more lines to see... Thanks